### PR TITLE
[webapi] Update manual tests to bdd tests for Canvas

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 3, Delete 0
Unit test Platform: Crosswalk Project for Android 15.44.374
Unit test result summary: Pass 3, Fail 0, Blocked 0

### CODE REVIEW

File: webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<widget id='http://tizen.org/test/html5-canvas-app' xmlns='http://www.w3.org/ns/widgets' xmlns:tizen='http://tizen.org/ns/widgets'>
```
Comment (Honry) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml]: What does this file use for?

File: webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5 [hunk 2]
```diff
@@ -0,0 +1 @@
+5fdc658cc7b9f789241cc98e69bb42d8
```
Comment (Honry) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5]: What about the baseline for arm device?

File: webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml (hunk 1 above)
Comment (JianfengXu) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml]: @Honry this file is the subapp config file for Tizen.

File: webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5 (hunk 2 above)
Comment (JianfengXu) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5]: @Honry I think all the baseline need be provided by nightly tester.

Comment (Honry) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5]: You can ask help from Li, Jun

File: webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml (hunk 1 above)
Comment (Honry) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml]: We don't need to consider the Tizen platform now.

Comment (JianfengXu) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/bdd.config.xml]: OK, and I will create a PR to delete  bdd.config.xml in other BDD sub apps.

File: webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5 (hunk 2 above)
Comment (JianfengXu) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5]: I think it is better that upload the baseline data after all the bdd tests are completed in webapi.

Comment (Honry) [inline webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5]: You can ask for the baseline by weekly, that tests can be tested smoothly in next release. I will merge it as-is.

**Pull request merged**